src/material/tree/tree.scss

```diff
@@ -9,7 +9,6 @@ $mat-node-height: 48px;
   align-items: center;
   min-height: $mat-node-height;
   flex: 1;
-  overflow: hidden;
```

Comment:
It would be good to see if this changes any screenshots in Google by running a global presubmit with just this line changed

Reply:
Changed 2 screenshots, but one was an improvement in my opinion and the other was a very nit box-shadow diff of some kind. Nothing major.